packages/vx-demo/src/sandboxes/vx-tooltip/Example.tsx

```diff
@@ -96,36 +88,30 @@ export default function Example({ width, height, showControls = true }: TooltipP
             <div
               className="position-indicator"
               style={{
-                width: positionIndicatorSize,
-                height: positionIndicatorSize,
-                transform: `translate(${(tooltipData?.containerX ?? 0) -
-                  positionIndicatorSize / 2}px, ${(tooltipData?.containerY ?? 0) -
+                transform: `translate(${tooltipLeft - positionIndicatorSize / 2}px, ${tooltipTop -
                   positionIndicatorSize / 2}px)`,
               }}
             />
             <div
               className="crosshair horizontal"
-              style={{ transform: `translateY(${tooltipData?.containerY ?? 0}px)` }}
+              style={{ transform: `translateY(${tooltipTop}px)` }}
             />
             <div
               className="crosshair vertical"
-              style={{ transform: `translateX(${tooltipData?.containerX ?? 0}px)` }}
+              style={{ transform: `translateX(${tooltipLeft}px)` }}
             />
-            <TooltipWrapper>
-              <TooltipComponent
-                key={Math.random()} // needed for bounds to update correctly
-                left={(tooltipLeft ?? 0) + (detectBounds ? 0 : 10)}
```

Comment:
noting that we got to remove this conditional `10` by making the `Tooltip` APIs consistent.